### PR TITLE
CI: add build docker test

### PR DESCRIPTION
**Pull request by ewfian**
Some changes may accidentally destroy the docker build(see: https://github.com/FlowiseAI/Flowise/issues/217), a CI check is added here to ensure that problems are found in the first place

Additional changes:
- [x] Fix trigger branch.
- [x] Optimize dockerfile, see: https://github.com/FlowiseAI/Flowise/issues/250#issuecomment-1576784879.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -10,8 +10,7 @@ RUN apk add --update libc6-compat
 WORKDIR /usr/src/packages
 
 # Copy root package.json and lockfile
-COPY package.json ./
-COPY yarn.lock ./
+COPY package.json yarn.loc[k] ./
```
[the remark this answers is not in the document]
Comment (HenryHengZJ) [reply Dockerfile:13]: Oh just saw your comment there. Ignore me 🤣

File: Dockerfile [hunk 2]
```diff
@@ -5,13 +5,12 @@
 # docker run -d -p 3000:3000 flowise
 
 FROM node:18-alpine
-RUN apk add --update libc6-compat
+RUN apk add --update libc6-compat python3 make g++
```
Comment (ewfian) [inline Dockerfile:8]: Are these three runtime dependencies? It can be compiled successfully even without adding.

Comment (HenryHengZJ) [reply Dockerfile:8]: here - https://github.com/FlowiseAI/Flowise/issues/241

**Comment by goldzulu** [comment]
@HenryHengZJ 

The line below seems to be a typo?

COPY package.json yarn.loc[k] ./. 

Shouldn't it be 
COPY package.json yarn.lock ./. 

Or is this intentional?

**Comment by ewfian** [comment]
@goldzulu It was written on purpose, see:  https://github.com/FlowiseAI/Flowise/issues/250#issuecomment-1576784879 and https://stackoverflow.com/a/70096420

**Pull request merged**